sonic-chassisd/tests/test_chassisd.py

```diff
@@ -703,6 +706,8 @@ def test_daemon_run_linecard():
        daemon_chassisd.run()
 
 def test_chassis_db_cleanup():
+    # Fix Me
+    return
```

Comment:
?

Reply:
This has dependency on is_smartswitch  defined in "https://github.com/sonic-net/sonic-platform-common/pull/454" to be committed.  Will unto this once this PR is committed.

Reply:
restored